integration/gethnetwork/geth_network.go

```diff
@@ -133,11 +132,8 @@ func NewGethNetwork(portStart int, websocketPortStart int, gethBinaryPath string
 	timestamp := strconv.FormatInt(time.Now().UnixMilli(), 10)
 	buildDir := path.Join(basepath, buildDirBase, timestamp)
 	// We create a data directory for each node.
-	nodesDir, err := ioutil.TempDir("", timestamp)
+	nodesDir := os.TempDir()
```

Comment:
@otherview I think we need to keep the timestamp here somehow, otherwise the logs for the sims will be completely jumbled and it'll be difficult to find the logs of the latest build specifically.